scripts/spell_check/spelling.dat

```diff
@@ -2390,6 +2390,8 @@ dosent':doesn't
 dosen't:doesn't
 doub:doubt
 doulbe:double
+double click:double-click
+doubleclick:double-click
```

Comment:
just give it a try with the star, it should make it.
thanks a lot for the effort